include/z64camera.h

```diff
@@ -1160,7 +1292,46 @@ typedef struct {
 } CamColChk; // size = 0x28
 
 typedef struct {
-    /* 0x000 */ char paramData[0x50];
+    /* 0x000*/ union {
+        Normal1 norm1;
+        Normal2 norm2;
```

Comment:
you should be able to typedef this outside of the struct so that we dont have this large definition in the camera struct itself